tox.ini

```diff
@@ -33,10 +33,12 @@ passenv= ProgramFiles LOGNAME USER LNAME USERNAME HOME USERPROFILE
 
 [testenv:lint]
 deps =
+    black
     flake8
     isort
     mypy
 commands =
+    black --target-version py27 --check --diff .
```

Comment:
Oh, I actually missed this. Why `py27`?

Reply:
@nir0s I think because Python 2.7 is still supported and hence no language features that need more recent versions of Python can be used. @jdufresne ?

Reply:
Yup. Once EOL Pythons are dropped, this can be bumped to `py36`.